.changeset/curvy-worms-invite.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/accessible-button': patch
+---
+
+keep role attribute of the accessibility button even when disabled
```

Comment:
Perhaps we want to be more generic here.
I'm suggesting this since I got a similar comment from @emmenko in a past PR.

```suggestion
fix a bug with accessibility attributes when using the `as` property
```

Reply:
Thank you.
 updated here: https://github.com/commercetools/ui-kit/pull/2262/commits/8efe0f98487925bae033392e5de1d8476b01c8c0